V2/server/model/dbqueries.js

```diff
@@ -41,5 +41,25 @@ class Model {
       throw err;
     }
   }
+
+  async update(columns, clause, values) {
+    const query = `UPDATE ${this.table} SET ${columns} WHERE ${clause} returning *`;
+    try {
+      const { rows } = await this.pool.query(query, values);
+      return rows[0];
+    } catch (err) {
+      throw err;
+    }
+  }
+
+  async delete(clause, values) {
+    const query = `DELETE FROM ${this.table} WHERE ${clause} returning *`;
```

Comment:
Can you define `query` as a global variable ... such that you don't repeat defining it even with     `const query = `UPDATE ${this.table} SET ${columns} WHERE ${clause} returning *`;
`